api/app/signals/apps/signals/tasks.py

```diff
@@ -63,19 +64,18 @@ def anonymize_reporter(reporter_id):
     except Reporter.DoesNotExist:
         log.warning(f"Reporter with ID #{reporter_id} does not exists")
     else:
-        reporter.anonymize()
-
-        changed = []
-        if reporter.email:
-            changed.append('email')
-        if reporter.phone:
-            changed.append('telefoonnummer')
-        if reporter.is_anonymized:
-            changed.append('anoniem')
-        text = 'Vanwege de AVG zijn de volgende gegevens van de melder ' \
-               'geanonimiseerd: {}'.format(', '.join(changed))
-
-        Signal.actions.create_note(data={
-            'text': text,
-            'created_by': None  # This wil show as "SIA systeem"
-        }, signal=reporter.signal)
+        if not reporter.is_anonymized:
+            reporter.anonymize()
+
+            changed = []
+            if reporter.email_anonymized:
```

Comment:
Either of these could already have been done (anonymizing email or phone numbers). But that is a nitpick that need not be addressed.